### PR TITLE
Use sample application setup script for build and bundle

### PR DESCRIPTION
**Pull request by cheng-kevin**
*Issue #, if available:*
* Need to discuss if we want customers of this action to be able to install gazebo version separately from ROS, or always use the default version.
* Also, when multiple ROS versions are supported per branch, should we just default to installing the latest supported distro?

*Description of changes:*
Use vanilla ubuntu image instead of rostooling image.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/test-ros2.yml [hunk 1]
```diff
@@ -17,15 +17,15 @@ jobs:
     
     strategy:      
       matrix:
```
Comment (ojasjoshi) [inline .github/workflows/test-ros2.yml:19]: nit: it doesn't look like we need a matrix in this workflow

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -17,36 +17,20 @@ jobs:
     
     strategy:      
       matrix:
-        distro: ['kinetic', 'melodic']
```
Comment (ojasjoshi) [inline .github/workflows/test.yml:20]: same as the other workflow, we can drop the matrix

**Comment by ojasjoshi** [comment]
Per our README, "This action will build and bundle your AWS RoboMaker Sample Application package". I don't see any use of having options for ROS_DISTRO, GAZEBO_VERSION and ROS_ENV_VARIABLES. ALL the dependencies will be managed by the setup script (including sourcing ROS env variables and gazebo). Unless we want to support running sample applications for multiple ROS distros within the same branch (which we currently don't) in future. In that case, we can have a setup script per ROS distro in the sample application repos and expose the option of ROS_DISTRO in this action to use the appropriate setup script.

**Comment by ojasjoshi** [comment]
Per our README, "This action will build and bundle your AWS RoboMaker Sample Application package". I don't see any use of having options for ROS_DISTRO, GAZEBO_VERSION and ROS_ENV_VARIABLES. ALL the dependencies will be managed by the setup script (including installing ROS and Gazebo). 

If we want to support running sample applications for multiple ROS distros within the same branch (which we currently don't) in future, we can have a setup script per ROS distro in the sample application repos and expose the option of ROS_DISTRO in this action to use the appropriate setup script.

**Comment by cheng-kevin** [comment]
The GAZEBO_VERSION may still be required, as we have other actions that depend on it.
`'travis/hello-world/${{ steps.robot_ws_build.outputs.ros-distro }}/${{ steps.robot_ws_build.outputs.gazebo-version }}/`

**Review by ojasjoshi — APPROVED** [review]

File: .github/workflows/test-ros2.yml [hunk 3]
```diff
@@ -43,7 +43,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: aws-robotics/aws-robomaker-sample-application-helloworld
-        ref: ros2
+        ref: ros2_foxy
```
Comment (cheng-kevin) [inline .github/workflows/test-ros2.yml]: fix this

**Pull request merged**